examples/10-tuning.py

```diff
@@ -0,0 +1,248 @@
+"""
+Parameter tuning for range-separated models
+===========================================
+
+.. currentmodule:: torchpme
+
+We explain and demonstrate parameter tuning for Ewald and PME
+"""
+
+# %%
+
+from time import time
+
+import ase
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import numpy as np
```

Comment:
Shouldn't we try to be pure torch or is there a reason why we need numpy in this example.